deploy/charts/rig-platform/templates/ingress.yaml

```diff
@@ -4,9 +4,12 @@ kind: Ingress
 metadata:
   name: {{ include "rig-platform.fullname" . }}
   labels: {{ include "rig-platform.labels" . | nindent 4 }}
+  annotations:
   {{- with .Values.ingress.annotations }}
-  annotations: {{ toYaml . | nindent 4 }}
+     {{ toYaml . | nindent 4 }}
   {{- end }}
+    cert-manager.io/cluster-issuer: "cloud-dns-issuer"
+    kubernetes.io/ingress.class: "nginx"
```

Comment:
I guess the annotations here, should be set in the provided values instead. Both of these highly depend on the environment in which rig is deployed.